packages/consts/src/regexs.ts

```diff
@@ -74,7 +76,11 @@ const GITHUB_REGEX_STR = '[a-z\\d](?:[a-z\\d]|-(?=[a-z\\d])){0,38}';
 export const TWITTER_REGEX = /^@[a-z0-9_]{1,15}$/i;
 export const GITHUB_REGEX = new RegExp(`^${GITHUB_REGEX_STR}$`, 'i');
 
-// @TODO: Discontinue usage of this regexps, in favor of HTTP_URL_REGEX
+export const LINKEDIN_PROFILE_REGEX = /^(https?:\/\/)?(www\.)?linkedin.com\/in\/([A-Za-z0-9]+)\/?$/;
```

Comment:
Important: There is another format for companies (ie: linkedin.com/company/apifytech/) which this regex doesn't match

Reply:
Good catch, resolved